knotx-core/knotx-repository/src/main/java/com/cognifide/knotx/repository/RepositoryConfiguration.java

```diff
@@ -149,6 +154,14 @@ public void setType(RepositoryType type) {
             this.type = type;
         }
 
+        public JsonObject getClientOptions() {
+            return clientOptions;
+        }
+
+        public void setClientOptions(JsonObject clientOptions) {
```

Comment:
Client options are being set at 44 line of RepositoryConfiguration:
repositoryMetadata.clientOptions = item.getJsonObject("client.options", new JsonObject());
All the setters of RepositoryConfiguration.RepositoryMetadata are used in UnitTests only